### DISCUSSION TITLE
Handover and mobility features

### DISCUSSION THREAD

**Discussion by arman-maghsoudnia** [srsran/srsRAN_Project#277] [General Help] [answered]
Hi community,

Thanks for your amazing work! I have some questions about the handover feature availablities. 

I see that in the documentation, handover is on the roadmap. And I also see the thread #196, saying it would be released very recently. So I'm wondering if there is any update about when you will publish it.

Thanks again for your cool project and help!

**Comment by ismagom** [comment, marked as answer]
We are currently expecting to release end of year
